parser_library/src/workspaces/workspace.cpp

```diff
@@ -842,7 +842,7 @@ bool workspace::load_config(config::proc_grps& proc_groups,
             if (!pg.asm_options.valid())
                 config_diags_.push_back(
                     diagnostic_s::error_W0005(proc_grps_file->get_location(), pg.name, "processor group"));
-            if (!pg.preprocessor.valid())
+            if (std::any_of(pg.preprocessors.begin(), pg.preprocessors.end(), [](const auto& x) { return !x.valid(); }))
                 config_diags_.push_back(diagnostic_s::error_W0006(proc_grps_file->get_location(), pg.name));
```

Comment:
Do you think it might be useful to give the user some information about which preprocessor is invalid? It might be more useful if there is more than just the current 2 preprocessors, of course.